AzureCalling/External/Calling/CallingContext.swift

```diff
@@ -469,6 +478,10 @@ extension CallingContext: CallDelegate {
             if let userIdentifier = participant.identifier.stringValue {
                 participant.delegate = self.participantsEventsAdapter
                 self.remoteParticipants.append(forKey: userIdentifier, value: participant)
+                if participant.videoStreams.contains(where: { $0.mediaStreamType == .screenSharing }) {
```

Comment:
Is this necessary? Would this be duplicated with the function inside `onVideoStreamsUpdated`?

Reply:
This would be duplicate function, but I think because there is logic in removeRemoteParticipants to remove the `currentScreenSharingParticipant` there should also be logic to add it when addRemoteParticipants.

But I will remove the code here and let `onVideoStreamsUpdated` handle to check for screen sharing